feat: integrate ipfs-repo-migrations tool (#2527)
ipfs/js-ipfs: README.md

const node = await IPFS.create({ repo: '/var/ipfs/data' })
```

##### `options.repoAutoMigrate`

| Type | Default |
|------|---------|
| boolean | `true` |

> `js-ipfs` comes bundled with tool that automatically migrate the version of your IPFS repository when new version is available.

Reviewer: ```suggestion
`js-ipfs` comes bundled with a tool that automatically migrates your IPFS repository when a new version is available.
```

> **For tools that build on top of `js-ipfs` and run mainly in the browser environment, be aware that disabling automatic

Reviewer: ```suggestion
**For apps that build on top of `js-ipfs` and run in the browser environment, be aware that disabling automatic
```
